src/tests/compiler/module/importDeclarationTests.ts

```diff
@@ -707,10 +707,19 @@ describe(nameof(ImportDeclaration), () => {
         it("should work for default with named imports", () => {
             doTest(`import bar, {test} from 'foo'`, {
                 defaultImport: "bar",
-                moduleSpecifier: "\'foo\'",
+                moduleSpecifier: "foo",
                 namedImports: [{ name: "test", alias: undefined }],
                 namespaceImport: undefined
             });
         });
     });
+
+    describe(nameof<ImportDeclaration>(n => n.getStructure) + " compatibiltiy with " +
```

Comment:
this test show the problem

Reply:
Thanks! I removed this though because it's superfluous. The other tests above ensure the problem won't happen and I'd prefer all tests just test one thing rather than testing multiple aspects at once.